config/v1.4/calico.yaml

```diff
@@ -177,7 +198,10 @@ metadata:
 spec:
   scope: Cluster
   group: crd.projectcalico.org
-  version: v1
+  versions:
```

Comment:
Using `version:` is deprecated, so I changed this according to the documentation https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition